src/olympia/amo/tests/test_amo_utils.py

```diff
@@ -145,9 +145,9 @@ def test_non_ascii_filename(self):
 
     def test_non_ascii_content(self):
         fn = os.path.join(self.tmp, 'somefile.txt')
-        with self.stor.open(fn, 'w') as fd:
+        with self.stor.open(fn, 'wb') as fd:
             fd.write(u'Ivan Krsti\u0107.txt'.encode('utf8'))
-        with self.stor.open(fn, 'r') as fd:
+        with self.stor.open(fn, 'rb') as fd:
             assert fd.read().decode('utf8') == u'Ivan Krsti\u0107.txt'
```

Comment:
tbh I went with the simplest change that didn't affect how the test was written. It was broken because Python 3 now reads the file in unicode by default, so I changed it to binary to avoid changing anything in the test iself.

Reply:
wfm